.gitignore

```diff
@@ -6,6 +6,7 @@ sites/*
 !sites/.gitkeep
 modules/*
 !modules/mod_ginger*
+modules/*/translations/template
```

Comment:
Not necessary because we already ignore `*.pot`

Reply:
Without it I did see the template directories in the "files to add".
Maybe the `!modules/...` influences the other rules.

Reply:
I could see requests to merge the pot files, could also be dependent on the git version.

Reply:
This is what I get if I remove this line from the .gitignore:

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	modules/mod_ginger_admin/translations/template/
	modules/mod_ginger_auth/translations/template/
	modules/mod_ginger_base/translations/template/
	modules/mod_ginger_edit/translations/template/
	modules/mod_ginger_foundation/translations/template/
	modules/mod_ginger_rdf/translations/template/
	modules/mod_ginger_remark/translations/template/
	modules/mod_ginger_search/translations/template/
```

Reply:
Ah, that’s because your `msgmerge.sh` creates *.po files in those directories (which are not ignored).

Reply:
Ah, it did create some garbage in those directories with an older version.  Removed the .gitignore rule.